.github/workflows/ci.yml

```diff
@@ -33,14 +33,10 @@ jobs:
     needs: [build-and-export-test-all-docker]
     steps:
       - uses: actions/checkout@v2
-      - name: Load Docker
```

Comment:
Can you verify if the PR changes work as expected by triggering a pipeline with `build-and-export-test-all-docker` generating the docker image?

Reply:
To verify this, I ran
https://github.com/tgonzalezorlandoarm/parsec/actions/runs/6800961661

That is based on this commit:
https://github.com/tgonzalezorlandoarm/parsec/commit/9639bc9845962578240223b998834b35900377e2

Reply:
Thanks